login/service.go

```diff
@@ -150,6 +200,29 @@ func (keycloak *KeycloakOAuthProvider) Perform(ctx *app.LoginLoginContext, confi
 			"user_name":      usr.Email,
 		}, "local user created/updated")
 
+		// new user for WIT
+		if newUser {
+			err = keycloak.remoteWITService.CreateWITUser(ctx, ctx.RequestData, usr, identity, WITEndpointUserProfile, identity.ID.String())
+			if err != nil {
+				log.Error(ctx, map[string]interface{}{
+					"err":         err,
+					"identity_id": identity.ID,
+					"username":    identity.Username,
+				}, "unable to create user in WIT ")
+				// let's carry on instead of erroring out ?
```

Comment:
Yeah. Don't fail if something wrong with WIT. Log the problem and proceed. Auth should not fail if WIT is down. The user will be updated in WIT in the next login.